File: app/src/main/java/com/nusretozates/wake_up/Utils/RandomStringGenerator.java
package com.nusretozates.wake_up.Utils;

import java.util.Random;

public class RandomStringGenerator
{

    Random generator = new Random();


    public String generate(int size)
    {
        StringBuilder builder = new StringBuilder();
        for(int i = 0;i<size;i++)
        {
            char randomletter = (char) (generator.nextInt(26) + 97);
            builder.append(randomletter);
        }

        return builder.toString();


    }


}

File: app/src/main/java/com/nusretozates/wake_up/Utils/RandomStringGenerator.kt
package com.nusretozates.wake_up.Utils

import java.util.*

class RandomStringGenerator {

    internal var generator = Random()


    fun generate(size: Int): String {
        val builder = StringBuilder()
        for (i in 0 until size) {
            val randomletter = (generator.nextInt(26) + 97).toChar()
            builder.append(randomletter)
        }

        return builder.toString()


    }


}
